src/contents/ui/main.qml

```diff
@@ -141,6 +165,38 @@ PlasmoidItem {
                     visible: player.artUrl
                     source: player.artUrl
                     fillMode: Image.PreserveAspectFit
+                    MouseArea {
+                        id: coverMouseArea
+                        anchors.fill: parent
+                        cursorShape: player.canRaise ? Qt.PointingHandCursor : Qt.ArrowCursor
+                        onClicked: {
+                            if (player.canRaise) player.raise()
+                        }
+                        hoverEnabled: true
+                    }
+                    PlasmaComponents3.ToolTip {
+                        id: raisePlayerTooltip
+                        anchors.centerIn: parent
+                        text: player.canRaise ? i18n("Bring player to the front") : i18n("This player can't be raised")
+                        width: parent.width - Kirigami.Units.largeSpacing
+                        contentItem: PlasmaComponents3.Label {
+                            text: raisePlayerTooltip.text
+                            font.pointSize: 16
+                            font.weight: Font.Bold
+                            wrapMode: Text.Wrap
+                            horizontalAlignment: Text.AlignHCenter
+                            // prevents tooltip from stealing the cursor
+                            MouseArea {
+                                anchors.centerIn: parent
+                                width: raisePlayerTooltip.width
+                                height: raisePlayerTooltip.height
+                                propagateComposedEvents: true
+                                cursorShape: Qt.PointingHandCursor
+                            }
+                        }
+                        visible: coverMouseArea.containsMouse
+                        delay: 0
+                    }
```

Comment:
I prefer the standard PlasmaComponents3.Tooltip, both for the look and consistency with the other tooltips, something like this should be fine:
```qml
PlasmaComponents3.ToolTip {
      id: raisePlayerTooltip
      anchors.centerIn: parent
      text: player.canRaise ? i18n("Bring player to the front") : i18n("This player can't be raised")
      visible: coverMouseArea.containsMouse
}
```

Reply:
Done